clinvar/src/main/scala/org/broadinstitute/monster/etl/clinvar/models/output/SCVTraitSet.scala

```diff
@@ -14,12 +14,14 @@ import upack.Msg
   *                                       includes this set, if the set was packaged
   *                                       directly under the observation
   * @param `type` common type of the trait collection
+  * @param scvTraitIds the IDs of the SCVTraits that make up the Trait Set
   */
 case class SCVTraitSet(
   id: String,
   clinicalAssertionId: Option[String],
   clinicalAssertionObservationId: Option[String],
-  `type`: Option[String]
+  `type`: Option[String],
+  scvTraitIds: Array[String]
```

Comment:
```suggestion
  clinicalAssertionTraitIds: Array[String]
```
And you can drop `clinicalAssertionId` and `clinicalAssertionObservationId`